specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/common/2022-01-15-preview/parameters.json

```diff
@@ -0,0 +1,62 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-01-15-preview",
+    "title": "Common types and definitions",
+    "description": "Definitions and parameters common for all resource types of KubernetesConfiguration RP"
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {},
+  "definitions": {},
+  "parameters": {
+    "ClusterRpParameter": {
+      "name": "clusterRp",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "Microsoft.ContainerService",
+        "Microsoft.Kubernetes"
+      ],
+      "description": "The Kubernetes cluster RP - either Microsoft.ContainerService (for AKS clusters) or Microsoft.Kubernetes (for OnPrem K8S clusters).",
+      "x-ms-parameter-location": "method"
+    },
+    "ClusterResourceNameParameter": {
```

Comment:
>ClusterResourceNameParameter [](http://example.com/codeflow?start=5&length=28)

Same thing here: x-mx-enum was removed w.r.t. the previous API version (2022-01-01-preview).

Reply:
I am not seeing any difference between my parameters and the ones from previous version 2022-01-01. Can you provide more detail on what this is referring to?

Reply:
The two parameter definitions in parameters.json had this removed:
```
      "x-ms-enum": {
        "name": "ExtensionsClusterRp",
        "modelAsString": true
      },
```
from ClusterRpParameter and this
```
      "x-ms-enum": {
        "name": "ExtensionsClusterResourceName",
        "modelAsString": true
      },
```
from ClusterResourceNameParameter. As I said, I don't think it affects the REST layer, but it affects SDK generation.

Reply:
Ok I see there were some changes made after I created this fork. I will update these properties